src/new-components/heading/heading.module.css

```diff
@@ -21,6 +21,8 @@
 
 h1.heading {
     font-size: var(--reactist-font-size-header);
+    font-weight: var(--reactist-font-weight-strong);
+    line-height: 43px;
```

Comment:
I haven't introduced variables for these as they are quite specific. Let me know if you feel they should have their own variables though.